setup.py

```diff
@@ -5,7 +5,7 @@
 
 setup(
     name='roman',
-    version='3.3.dev0',
+    version='3.3.dev1',
```

Comment:
Please remove the version change. The person who does a release makes all necessary changes.